docs/clients/athena.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: client
+category: clients
+name: Athena
+package: async-aws/athena
+---
+
+## Usage
+
+### List Databases
+
+```php
+use AsyncAws\Athena\AthenaClient;
+use AsyncAws\Athena\Input\ListDatabasesInput;
+
+$athena = new AthenaClient();
+
+$result = $athena->listDatabases(new ListDatabasesInput([
+    'CatalogName' => 'my_catalog'
+    'NextToken' => 'next-1045'
+    'MaxResults' => 20
+]));
+
+$databaseList = $result->getDatabaseList();
+foreach ($result->getDatabaseList() as $database) {
+    echo 'Database name : ' . $database->getName();
+    echo 'Database description : ' . $database->getDescription();
+    echo 'Database parameter : '.PHP_EOL;
+    print_r($database->getParameters());
+}
+
+echo 'Next token : ' . $result->getNextToken(). "\n";
+```
+more information [listDatabases](https://docs.aws.amazon.com/athena/latest/APIReference/API_ListDatabases.html)
+
+### Query to Amazon Athena
+
+```php
+use AsyncAws\Athena\AthenaClient;
+use AsyncAws\Athena\Input\StartQueryExecutionInput;
+use AsyncAws\Athena\Input\DescribeTableInput;
+use AsyncAws\Athena\ValueObject\QueryExecutionContext;
+use AsyncAws\Athena\ValueObject\ResultConfiguration;
+use AsyncAws\Athena\ValueObject\EncryptionConfiguration;
+use AsyncAws\Athena\ValueObject\AclConfiguration;
+use AsyncAws\Athena\ValueObject\ResultReuseByAgeConfiguration;
+use AsyncAws\Athena\Input\GetQueryExecutionInput;
+use AsyncAws\Athena\Input\GetQueryResultsInput;
+use AsyncAws\Athena\ValueObject\Row;
+use AsyncAws\Athena\ValueObject\Datum;
+
+$athena = new AthenaClient();
+
+// Submits a sample query to Amazon Athena and returns the execution ID of the query.
+$startQueryResult = $athena->startQueryExecution(new StartQueryExecutionInput([
+        'QueryString' => 'select * from my_table limit 3 where uid =: param1', // REQUIRED
+        'ClientRequestToken' => 'Unique-caSe-sensitive-0011'
+        'QueryExecutionContext' => new QueryExecutionContext([
+             'Database' => 'iad_database_name', // REQUIRED
+             'Catalog' => 'iad_catalog_name'
+        ]),
+        'ResultConfiguration' => new ResultConfiguration([
+            'OutputLocation' => 's3://output_bucket_Location', // REQUIRED
+            'ExpectedBucketOwner' => 'iad-account',
+            'EncryptionConfiguration' => new EncryptionConfiguration([
+                'EncryptionOption' => 'SSE_S3', // REQUIRED SSE_S3|SSE_KMS|CSE_KMS
+                'KmsKey' => '<string>',
+            ]),
+            'AclConfiguration' => new AclConfiguration([
+                'S3AclOption' => 'BUCKET_OWNER_FULL_CONTROL',
+            ])
+        ]),
+        'ResultReuseConfiguration' => new ResultReuseByAgeConfiguration([
+            'Enabled' => true || false,
+            'MaxAgeInMinutes' => <integer>,
+        ])
+        'WorkGroup' => 'iadinternational',
+        'ExecutionParameters' => ['param1']
```

Comment:
I suggest exposing a minimal "real world" example without optional settings (ie. KMS)
and using real name